### PR TITLE
Bind wlr_subsurface_from_wlr_surface and wlr_surface_is_subsurface

### PR DESCRIPTION
**Pull request by dfangx**
Required by https://github.com/riverwm/river/pull/568

### CODE REVIEW

**Comment by dfangx** [comment]
@novakne could you give this a review?

**Comment by dfangx** [comment]
@novakne could you give this a review?

**Comment by novakne** [comment]
LGTM, although maybe I'll move `wlr_subsurface_from_wlr_surface` in Surface since it takes a Surface as argument. @ifreund would be the one who decides on this though.

File: src/types/surface.zig [hunk 1]
```diff
@@ -234,4 +237,7 @@ pub const Subsurface = extern struct {
     },
 
     data: usize,
+
+    extern fn wlr_subsurface_from_wlr_surface(surface: *Surface) *wlr.Subsurface;
```
Comment (ifreund) [inline src/types/surface.zig:241]: >  Get a subsurface from a surface. Can return NULL if the subsurface has been destroyed.
```suggestion
    extern fn wlr_subsurface_from_wlr_surface(surface: *Surface) ?*wlr.Subsurface;
```

**Review by ifreund — CHANGES_REQUESTED** [review]
Looks good to me aside from comment.

**Pull request merged**